### ISSUE TITLE
Is there any way to use an encoder that supports HWA via QSV and NVENC?

### ISSUE THREAD

**Issue by dr04e606** [vkohaupt/vokoscreenNG#226]
Lately, I've been trying to use vokoscreenNG on Windows 10. 
Upon installing, I noticed that it downloaded cisco/openh264. Sadly, it only supports software (CPU) encoding that is a bit too much for my fanless thin and light laptop.
Is there any way I can make vokoscreenNG use hardware-accelerated (GPU) encoder? As far as I know, ffmpeg supports HWA and is cross-platform. I was surprised to see that vokoscreenNG doesn't use ffmpeg. 
I'm not a developer, just a user, well, maybe a bit more advanced than your average Joe. Anyway, if there is no user-friendly way to make vokoscreenNG use a HWA encoder, consider this post a feature request.

**Comment by vkohaupt** [comment]
Hello, it's been half a year and I would like to reply here as well, after having received several inquiries privately.

H.264 is patent-bound, which means that a license fee of 20 cents must be paid for each installation.
It doesn't matter whether software or hardware is used, since Vokoscreen has to provide the codec or the interface.

Now everyone can calculate for themselves what hundreds of thousands of installations cost each year.

Here are two links in German. Fedora and opensuse will disable hardware acceleration.
https://www.golem.de/news/video-codecs-fedora-deaktivated-hardware-beschleunigung-wegen-patenten-2209-168588.html
and
https://linuxnews.de/2022/10/01/fedora-und-opensuse-deaktivieren-patentierte-codecs-in-mesa/

Possibly one could consider charging 1€ for each download. This should also be required for beta versions, but who is still testing? And there are additional costs, online shop, sales tax, tax consultant and possibly an employee who takes care of all these things.

In short, at the moment I see no possibility of implementing this in a reasonable financial framework.
